src/android/aidl/example-service/service-bindings.md

```diff
@@ -10,21 +9,25 @@ _birthday_service/aidl/com/example/birthdayservice/IBirthdayService.aidl_:
 }
 ```
 
-_Generated trait_:
+_out/soong/.intermediates/.../birthdayservice/IBirthdayService.rs_:
+
+<!-- dprint-ignore-start -->
 
 ```rust,ignore
-trait IBirthdayService {
-    fn wishHappyBirthday(&self, name: &str, years: i32) -> binder::Result<String>;
+pub trait IBirthdayService: binder::Interface + Send {
+  fn get_descriptor() -> &'static str where Self: Sized { "com.example.birthdayservice.IBirthdayService" }
+  fn r#wishHappyBirthday(&self, _arg_name: &str, _arg_years: i32) -> binder::Result<String>;
+  // ... more methods ...
```

Comment:
I don't think this change is an improvement. The original version was deliberately a cleaned up version of the generated trait so that it's easier for students to understand. I think directly showing the messier generated output is only going to be a distraction. What was your reasoning for changing this?

Reply:
Thank you, now that I see it again, I agree with you. I guess the difference surprised me while teaching the class, but you're right that it has a benefit to show the cleaned up version!

Reply:
I restored the old code and the PR now only contains the changes to the surrounding text.